### ISSUE TITLE
ipad issues

### ISSUE THREAD

**Issue by MartijnR** [kobotoolbox/enketo-core#6]
- question one border missing around options (very minor issue)
- question 2 label overlaps with images
- **question 3 not responsive** questions remain next to each other (overflow issue?)

**Comment by MartijnR** [comment]
let me know if any do not show up in enketo-core (locally) on ipad simulator please

**Comment by MartijnR** [comment]
Sometimes I can reproduce issue 2 on desktop in Chrome. Also has no background:

<img width="1006" alt="screen shot 2015-09-18 at 4 10 27 pm" src="https://cloud.githubusercontent.com/assets/627350/9972241/ce1326a2-5e1f-11e5-9da9-71bb92b5a3c1.png">

This moves the label to the correct location:

``` css
.touch .question.simple-select .option-wrapper > label {
    padding-left: 124px;
}
```

**Comment by pmusaraj** [comment]
This has mostly been fixed in commit 6c30b32

**Comment by MartijnR** [comment]
:+1:
